components/Collections/src/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -130,6 +130,11 @@ private set
     /// An <see cref="Action"/> that is called if an error occurs during data retrieval.
     /// </param>
     /// <seealso cref="IIncrementalSource{TSource}"/>
+#if NET8_0_OR_GREATER
+    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("This constructor is not AOT compatible and will be removed in a future version.")]
+#endif
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [Obsolete("This constructor is not AOT compatible and will be removed in a future version.")]
```

Comment:
Can we provide more here or a remark/example of how to convert the use of an existing use of this constructor to a valid one?